docs/releases/v2022.1.2.rst

```diff
@@ -0,0 +1,37 @@
+Gluon 2022.1.2
+==============
+
+Bugfixes
+--------
+
+ * Various build-errors which sporadously occur when building with a large thread-count have been fixed
```

Comment:
"sporadously" doesn't exist i think, please switch to "sometimes" or "sporadic"

Reply:
sporadically would be best fitting